### PR TITLE
Fix Solr install

### PR DESCRIPTION
**Pull request by Nuttymoon**
#### Which issue(s) this PR fixes:

Fixes:
- #326 
- #426 

#### Additional comments:

To make clear that you license your contribution under
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0) and that you give permission to [TOSIT](https://www.tosit.io/),
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [x] I hereby agree to grant [TOSIT](https://www.tosit.io/) a copyright license to use my contributions.

### CODE REVIEW

**Review by gboutry — CHANGES_REQUESTED** [review]
Can you regenerate the meta playbooks with the command `tdp playbooks` ?

File: tdp_lib_dag/ranger.yml [hunk 1]
```diff
@@ -76,6 +76,7 @@
 - name: ranger_solr_start
   depends_on:
     - ranger_solr_install
+    - ranger_admin_start
```
[the remark this answers is not in the document]
Comment (Nuttymoon) [reply tdp_lib_dag/ranger.yml:79]: I agree with @leopaul36 that every service has a soft dependency on Solr. But I do not recall a case where this is fatal.

[the remark this answers is not in the document]
Comment (Nuttymoon) [reply tdp_lib_dag/ranger.yml:79]: As for the `ranger_admin_start`, I thought that Solr needed to communicate with Ranger admin during init but after double checking, it's not the case.

Force pushing to remove the dependency.

**Review by rpignolet — COMMENTED** [review]
LGTM for Ansible code, for DAG I leave it to the experts

File: tdp_lib_dag/ranger.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (gboutry) [reply tdp_lib_dag/ranger.yml]: IMO, we should have the admin depend on solr.
We can depend on something that won't happen (noop, or not in topology). It's not a problem.

But, if we run a deployment with `ranger_admin_init` as target, should we expect solr installed and ready to work with ranger admin or not ?
If the answer is yes, ranger admin must depend on solr.
If the answer is no, ranger admin does not need to depend on solr.

[the remark this answers is not in the document]
Comment (Nuttymoon) [reply tdp_lib_dag/ranger.yml]: Works for me. Forced push to make `ranger_admin_start` depend on `ranger_solr_init`

**Review by gboutry — APPROVED** [review]

**Pull request merged**